.github/workflows/pr.yml

```
@@ -53,6 +57,7 @@ jobs:
           DEFI_SDK_API_URL: ${{ github.event.inputs.DEFI_SDK_API_URL || 'wss://api-v4.zerion.io/' }}
           ZERION_API_URL: ${{ github.event.inputs.ZERION_API_URL || 'https://zpi.zerion.io/' }}
           BACKEND_ENV: ${{ github.event.inputs.BACKEND_ENV || '' }}
+          FEATURE_PAYMASTER: ${{ github.event.inputs.FEATURE_PAYMASTER || 'on' }}
```

TODO before merging:
```suggestion
          FEATURE_PAYMASTER: ${{ github.event.inputs.FEATURE_PAYMASTER || '' }}
```